drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/renderers/ScenarioGridColumnRenderer.java

```diff
@@ -109,16 +111,20 @@ protected void applyBackgroundColor(ScenarioGridCell cell,
         }
     }
 
-    protected String getValueToShow(ScenarioGridCell scenarioGridCell){
+    protected String getValueToShow(ScenarioGridCell scenarioGridCell) {
         String rawValue = scenarioGridCell.getValue() != null ? scenarioGridCell.getValue().getValue() : null;
         return (rawValue != null && (scenarioGridCell.isList() || scenarioGridCell.isMap())) ? getCollectionString(rawValue, scenarioGridCell.isList()) : rawValue;
     }
 
     protected String getCollectionString(String jsonString, boolean isList) {
         try {
-            int size = -1;
-            String toFormat = isList ? "List(%s)" : "Map(%s)";
             JSONValue jsonValue = JSONParser.parseStrict(jsonString);
+            if (jsonValue instanceof JSONString) {
```

Comment:
@yesamer 
I think it would be clearer to and an "else" statement, or to create two different methods. Wdyt?

Reply:
@gitgabrio I added the else statement